pages/common/asciidoctor.md

```diff
@@ -9,12 +9,12 @@
 
 - Convert a specific `.adoc` file to HTML and link a CSS stylesheet:
 
-`asciidoctor {{[-a|--attribute]}} stylesheet {{path/to/stylesheet.css}} {{path/to/file.adoc}}`
+`asciidoctor {{[-a|--attribute]}} stylesheet={{path/to/stylesheet.css}} {{path/to/file.adoc}}`
```

Comment:
Wait this isn't an option. The = can stay. Does this command even work without =?

Reply:
I've tested it a bit and can confirm, that equal sign is mandatory here. Without it `asciidoctor` treats the stylesheet file as a source file that need to be converted, not a stylesheet file.

Reply:
You should change the page in that case

Reply:
OK, so we are leaving this as it is. Do I need to re-request a review from you? Now it shows that it still has a pending change request form you, @Managor.